setup.py

```diff
@@ -117,13 +119,22 @@ def ismac():
 except ModuleNotFoundError:
     ext_mods = []
 
-install_requirements = ['numpy>=1.7', 'six', 'paramz>=0.9.0', 'cython>=0.29']
-matplotlib_version = 'matplotlib==3.3.4'
-if sys.version_info < (3, 6):
-    install_requirements += ['scipy>=1.3.0,<1.5.0']
-    matplotlib_version = 'matplotlib==3.0.0'
-else:
-    install_requirements += ['scipy>=1.3.0']
+install_requirements = [
+    "numpy>=1.7",
+    "six",
+    "paramz>=0.9.0",
+    "cython>=0.29",
+    'scipy>=1.3.0,<1.5.0;python_version<"3.6"',
+    'scipy>=1.3.0;python_version>="3.6"',
+]
+
+matplotlib_version = [
+    'matplotlib==3.0.0;python_version<"3.6"',
+    'matplotlib==3.3.4;python_version>="3.6"',
+]
+
+install_requirements += matplotlib_version
```

Comment:
this makes matplotlib an unconditional dependency, which I don't think it was previously

Reply:
you are right.